internal/brokers/broker_test.go

```diff
@@ -68,11 +68,13 @@ func TestNewBroker(t *testing.T) {
 			if tc.wantErr {
 				configDir = filepath.Join(brokerConfFixtures, "invalid_brokers")
 			}
-			if tc.configFile != "" {
+			if tc.configFile == "-" {
+				tc.configFile = ""
+			} else if tc.configFile != "" {
```

Comment:
What about using a `ptrValue` here so that we don't have to use the special `-` and this logic?

Reply:
I don’t like the `ptrValue` for this. I prefer our existing pattern and this is what we are using in most of project. So let’s use that one :)